src/combobox/ComboBox.ts

```diff
@@ -385,7 +387,7 @@ export default class ComboBox extends ComboBoxBase<ComboBoxProperties> {
 			clearable ? v('button', {
 				'aria-controls': menuId,
 				bind: this,
-				classes: this.classes(css.clear),
+				classes: this.classes(css.clear, iconCss.icon, iconCss.times),
```

Comment:
yes and no, so `composes` *can* do that, BUT it creates other problems and duplicated code. The `icons.css` file is adapted from the `icomoon` output. Adding `composes` or similar would mean further changes to the standard output and the actual output would just be two classes anyway.